dist/index.html

```diff
@@ -124,8 +124,70 @@ <h3 class="text-c100 text-md py-3 font-bold">We Nourish</h3>
       Peoples Feedback
     </section>
 
-    <section class="work-style">
-      Work Style
+    <section class="work-style text-c600">
+      <div class="container">
+        <h2 class="text-center text-c100 font-bold">
+          How We <span class="text-c200 font-hairline underline">Work</span>
+        </h2>
+
+        <div class="showcase-row items-center">
+          <div class="text-center">
+            <div class="border-solid border-6 rounded-full border-c200">
```

Comment:
instead of rounded-full You can use 
{
border-radius: 50%;
    width: 150px;
    height: 150px;
}

or you can change the width and height like you want but use (50%)